README.md

```diff
@@ -1,2 +1,42 @@
 # elixir-base58-encode
 https://hex.pm/packages/b58
+
+
+This module provide the `encode/1` function which takes a binary and returns
+its base58 representation
+
+1. Add the package to you dependencies
+
+  ```
+  defp deps do
+    [b58: "~> 0.1.0"]
+  end
+  ```
+
+  and run `mix deps.get`
+
+2. Call the `encode` fundtion with a binary as parameter:
+
+ ```
+ > Base58Encode.encode("foo")
```

Comment:
is "foo" a "binary" or a "String"? 
see: https://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html#binaries-and-bitstrings